doc/builders/special/darwin-builder.section.md

```diff
@@ -1,11 +1,10 @@
-# darwin.builder {#sec-darwin-builder}
+# darwin.linux-builder {#sec-darwin-builder}
 
-`darwin.builder` provides a way to bootstrap a Linux builder on a macOS machine.
+`darwin.linux-builder` provides a way to bootstrap a Linux builder on a macOS machine.
 
 This requires macOS version 12.4 or later.
 
-This also requires that port 22 on your machine is free (since Nix does not
-permit specifying a non-default SSH port for builders).
+The builder runs on host port 31022 by default, you can change it by overriding `virtualisation.darwin-builder.hostPort`.
```

Comment:
```suggestion
The builder runs on host port 31022 by default.
You can change it by overriding `virtualisation.darwin-builder.hostPort`.
See the [example](#ex-darwin-linux-builder-flake).
```

We try to do one sentence per line. A single newline is not rendered as a line break.

Also down below (can't make a proper suggestion because it's not touched)

```diff
-## Example flake usage
+## Example flake usage {#ex-darwin-linux-builder-flake}
```

Reply:
Updated :+1: